src/main/java/duke/parser/TaskParser.java

```diff
@@ -0,0 +1,65 @@
+package duke.parser;
+
+import java.util.Optional;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+/**
+ * Parses Task into string and vice versa.
+ * @author Toh Li Yuan (A0255811H)
+ */
+public class TaskParser {
+
+    /**
+     * Formats an array of Task for local storage.
+     *
+     * @param tasks the array of Task to be stored.
+     * @return the formatted string to be stored.
+     */
+    public static String formatSave(Task[] tasks) {
```

Comment:
perhaps the name can be modified to make it sound like it returns a String. It kind of sounds like it returns null.